cmd/config/internal/commands/test/testdata/dataset2/mysql/Krmfile

```diff
@@ -0,0 +1,6 @@
+apiVersion: krm.dev/v1alpha1
```

Comment:
Can we make the testdata here a bit more understandable to others? At least name the different datasets with names that actually describe what they are meant for.

Reply:
Done.